### PR TITLE
Experiment

### PR DESCRIPTION
**Pull request by mnus2019**
link to GitHub repo

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -53,9 +53,11 @@ class BlogIndex extends React.Component {
...
           <section id="header">
             <div className="container">
-              <h1 id="logo">
-                <a href="/">{ siteTitle }</a>
-              </h1>
+              <h1 id="logo">{ siteTitle } 
+                <a href="https://github.com/junior-dev-struggle-bus/junior-dev-struggle-blog/issues/30" title ="GitHub"target="_blank"  
+                               
+                >BLOG</a>  
+                              </h1>
```
Comment (bhurstGH) [inline src/pages/index.js]: I think what we wanted here was the "Junior Dev Struggle Blog" followed by a GitHub icon that linked to the blog repo at https://github.com/junior-dev-struggle-bus/junior-dev-struggle-blog/

File: static/main.css [hunk 2]
```diff
@@ -107,7 +107,7 @@ input, select, textarea {
 
 	h1, h2, h3, h4, h5, h6 {
 		font-weight: 600;
-		text-transform: uppercase;
+		text-transform: camelcase;
```
Comment (bhurstGH) [inline static/main.css]: To my knowledge, I don't think there is a text transform for camel case. There is a capitalize, which capitalizes the first letter of each word, though.

Comment (mnus2019) [reply static/main.css]: hi dear brett,understood man,this gave more motivation to do more and work harder as fresh learner. In my local computer it's fine,but when i pull it in the github i got some errors dear brett.
I will do more to fix the errors as u have suggested brett or at the meetup dear man..thank you more ....

**Comment by nspilman** [comment]
Closing PR, as there is a more recent version. 
https://github.com/junior-dev-struggle-bus/junior-dev-struggle-blog/pull/66

**Pull request closed without merging**